pkg/cnbutils/env.go

```diff
@@ -0,0 +1,29 @@
+package cnbutils
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+)
+
+func CreateEnvFiles(utils BuildUtils, platformPath string, env []string) error {
+	envDir := filepath.Join(platformPath, "env")
+	err := utils.MkdirAll(envDir, 0755)
+	if err != nil {
+		return err
+	}
+
+	for _, e := range env {
+		eSplit := strings.SplitN(e, "=", 2)
+
+		if len(eSplit) != 2 {
```

Comment:
Theoretically, `KEY=` would be a valid variable. But this is probably just an edge case and not relevant for `cnbBuild`.

Reply:
`KEY=` will still end up in a slice of 2 elements, `["KEY", ""]`